flow/core/kernel/vehicle/traci.py

```diff
@@ -113,7 +113,10 @@ def initialize(self, vehicles):
                 self.__vehicles[veh_id] = dict()
                 self.__vehicles[veh_id]['type'] = typ['veh_id']
                 self.__vehicles[veh_id]['initial_speed'] = typ['initial_speed']
-                self.__vehicles[veh_id]["accel_without_noise"] = None
+                self.__vehicles[veh_id]["accel_no_noise_no_failsafe"] = None
+                self.__vehicles[veh_id]["accel_no_noise_with_failsafe"] = None
+                self.__vehicles[veh_id]["accel_with_noise_no_failsafe"] = None
+                self.__vehicles[veh_id]["accel_with_noise_with_failsafe"] = None
```

Comment:
Since there is a key existence check in all the get methods below. There is no need to initialize them here.